remove(docs): remove sending telemetry feature (#1661)
camunda/camunda-docs-manual: content/update/minor/721-to-722/_index.md

### If using Embedded Engine:

- You can keep using your Groovy version without any extra effort.

# Sending telemetry feature removed

Sending telemetry data has been introduced in Camunda `7.14.0+` (and `7.13.7`, `7.12.12`, `7.11.19`)
 and removed in Camunda `7.22.0`. The public API is marked as deprecated and emptied out.
 The telemetry reporter and scheduled timer and all related process engine configuration properties are removed.

>  The [diagnostic data][] is still being collected but not send by any mean.

Reviewer: Seems there's a missing link here.

Author: Discussed. The link is declared 4 rows below.

>  The [diagnostic data][] is still being collected but not send by any mean.

Reviewer: ```suggestion
 The [diagnostic data][] is still being collected but not sent by any mean.
```

  You can decide to share it with Camunda if requested in tickets or use it for your diagnostic purposes.

In previous Camunda versions - `7.21.0+`, `7.20.8+`, `7.19.15+`, reporting telemetry data is disabled by default.

[diagnostic data]: {{< ref "/user-guide/process-engine/diagnostics-data.md" >}}

## Configuration properties removed

> To clean up and refactor our source code, tte following process engine configuration properties have been remove.

Reviewer: ```suggestion
To clean up and refactor our source code, tte following process engine configuration properties have been removed.
```

> Please remove all of the occurrences of those properties, no matter of the setup that you are using

Reviewer: ```suggestion
Please remove all of the occurrences of those properties, regardless of the setup that you are using
```

(share or embedded process engine, pre-packaged or other distribution).
You need to remove the properties from your tests as well.

* `initializeTelemetry       `
* `telemetryReporterActivate `
* `telemetryReportingPeriod  `
* `telemetryReporterActivate `
* `telemetryRequestRetries   `
* `telemetryRequestTimeout   `

## Public API deprecation

The public API for configuring telemetry and fetching the telemetry configuration have been emptied out and marked as deprecated. 

> We recommend to remove the usage of the following API as the endpoints no longer do anything and is unnecessary.

Reviewer: ```suggestion
We recommend to remove the usage of the following API as the endpoints no longer do anything and their usage is unnecessary.
```
